content/_data/logo/singapore.yml

```diff
@@ -0,0 +1,8 @@
+---
+name: 'Singapore'
+url: '/images/logos/singapore/singapore.png'
```

Comment:
Maybe it is better to delete this as well. This is a logo of Cloud Native Jenkins from Jenkins World, and I am about using the original version for Cloud Native SIG

Reply:
singapore.yml removed. and images folder also removed.